py3-yarl.yaml

```diff
@@ -29,10 +37,28 @@ pipeline:
       expected-sha256: 566db86717cf8080b99b58b083b773a908ae40f06681e87e589a976faf8246bf
       uri: https://files.pythonhosted.org/packages/source/y/yarl/yarl-${{package.version}}.tar.gz
 
-  - name: Python Build
-    uses: python/build-wheel
-
-  - uses: strip
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      provides:
+        - py${{range.key}}-${{vars.pypi-package}}
+      runtime:
+        - py${{range.key}}-idna
+        - py${{range.key}}-multidict
```

Comment:
I noticed you dropped typing-extensions.
```

$ curl https://pypi.org/pypi/yarl/json | jq '.info | .version, .requires_dist' < out
"1.9.4"
[
  "idna >=2.0",
  "multidict >=4.0",
  "typing-extensions >=3.7.4 ; python_version < \"3.8\""
]
```
that says its ok to do that.
It'd be good to mention it in the commit message for others.